build-system/pr-check/build-targets.js

```diff
@@ -148,6 +149,17 @@ const targetMatchers = {
       file.startsWith('build-system/babel-config/')
     );
   },
+  [Targets.BUILD_SYSTEM]: (file) => {
+    if (isOwnersFile(file)) {
+      return false;
+    }
+
+    return (
+      file == 'build-system/tsconfig.json' ||
```

Comment:
For self-documentation and uniformity, I suggest explicitly listing `check-build-system.js` here instead of relying on the condition below to implicitly catch it.
```suggestion
      file == 'build-system/tasks/check-build-system.js' ||
      file == 'build-system/tsconfig.json' ||
```